complete-application/src/app/make-change-page/make-change-page.component.ts

```diff
@@ -0,0 +1,25 @@
+import { Component } from '@angular/core';
+import {CommonModule} from "@angular/common";
+import {FormsModule} from "@angular/forms";
+
+@Component({
+  selector: 'app-make-change-page',
+  standalone: true,
+  imports: [CommonModule, FormsModule],
+  templateUrl: './make-change-page.component.html',
+  styleUrls: ['./make-change-page.component.css']
+})
+export class MakeChangePageComponent {
+
+  amount = 0;
+
+  change: { total: number; nickels: number; pennies: number } | null = null;
+
+  makeChange() {
+    const total = this.amount;
```

Comment:
Please include quarters, here is an example in JSX

https://github.com/FusionAuth/fusionauth-quickstart-javascript-nextjs-web/blob/main/complete-application/src/components/MakeChangeForm.tsx